### ISSUE TITLE
Couldn't read comment file

### ISSUE THREAD

**Issue by tomqwu** [jenkinsci/ghprb-plugin#656]
I tested both scenarios of running the job on master and slave, both get this error
```
+ echo hello
+ cat commentFile.txt
hello
Setting status of a6425be4235ddb7ea9b438b271b7084f526ea81e to SUCCESS with url https://jenkins.example.com/job/Test-PRB/3/ and message: 'Jenkins job ran successful!
 '
Couldn't read comment file
java.io.FileNotFoundException: File 'commentFile.txt' does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1748)
	at org.jenkinsci.plugins.ghprb.extensions.comments.GhprbCommentFile.postBuildComment(GhprbCommentFile.java:42)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.commentOnBuildResult(GhprbBuilds.java:241)
	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:215)
	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:28)
	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:211)
	at hudson.model.Run.execute(Run.java:1772)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)

!!! Couldn't read commit file !!!

Finished: SUCCESS
```

**Comment by tomqwu** [comment]
Fixed by having 1.41 built from local, looking forward with with official 1.41 release

**Comment by kthacker7** [comment]
How did you add the comment file?
